### ISSUE TITLE
[BUG] error: unsupported tarball input attribute 'lastModified' when doing nix flake update

### ISSUE THREAD

**Issue by tibetsprague** [holochain/holochain#4113]
**Describe the bug**
Trying to do `nix flake update` and getting this error
`error: unsupported tarball input attribute 'lastModified'`

**Expected behavior**
It should work

**System information:**
 - OS: macOS Sonoma
 
    holochain-nix-versions.url =
      "github:holochain/holochain/?dir=versions/0_3";

**My complete flake.nix**

```
{
  description = "Template for Holochain app development";

  inputs = {
    p2p-shipyard.url = "github:darksoil-studio/p2p-shipyard";
    holochain-nix-versions.url =
      "github:holochain/holochain/?dir=versions/0_3";
    holochain-flake = {
      url = "github:holochain/holochain";
      inputs.versions.follows = "holochain-nix-versions";
    };

    nixpkgs.follows = "holochain-flake/nixpkgs";
    flake-parts.follows = "holochain-flake/flake-parts";
  };

  outputs = inputs@{ flake-parts, holochain-flake, ... }:
    flake-parts.lib.mkFlake {
      specialArgs.nonWasmCrates = [ "relay" ];
      inherit inputs;
    } {
      systems = builtins.attrNames holochain-flake.devShells;
      perSystem = { config, pkgs, system, inputs', ... }: {
        devShells.default = pkgs.mkShell {
          inputsFrom = [
            inputs'.p2p-shipyard.devShells.holochainTauriDev
            holochain-flake.devShells.${system}.holonix
          ];
          packages = [ pkgs.nodejs-18_x ];
        };
        devShells.androidDev = pkgs.mkShell {
          inputsFrom = [
            inputs'.p2p-shipyard.devShells.holochainTauriAndroidDev
            holochain-flake.devShells.${system}.holonix
          ];
          packages = [ pkgs.nodejs-18_x ];
        };
      };
    };
}
```

**Comment by ThetaSinner** [comment]
What Nix version are you using?

Please could you try upgrading to the latest version?

**Comment by nphias** [comment]
Bug resolved for me after upgrading to nix 2.23

**Comment by ThetaSinner** [comment]
This was an issue with Holochain upgrading to a newer Nix version and that changed the lock file format and broke things for people using an older Nix version. Upgrading is the solution and we have a ticket open on the new Holonix to look at testing against a minimum Nix version
